bindings/java/src/main/com/apple/foundationdb/ReadTransaction.java

```diff
@@ -210,6 +210,15 @@
 	 *  can provide speed improvements or efficiency gains based on the caller's
 	 *  knowledge of the upcoming access pattern.
 	 *
+	 * <p>
+	 *     When converting the result of this query to a list using `AsyncIterable.asList` with the `ITERATOR` streaming
```

Comment:
To make this really user-friendly, I think we should have the code blocks in these sections be made as links to the code being referenced.

Reply:
Agreed